draft-ietf-dnsop-svcb-https.md

```diff
@@ -652,10 +652,13 @@ See {{incomplete-response}} for possible optimizations of this procedure.
 ## General requirements
 
 Recursive resolvers MUST be able to convey SVCB records with unrecognized
-SvcParamKeys or malformed SvcParamValues.  Resolvers MAY treat the entire
+SvcParamKeys, and MAY treat the entire
 SvcParams portion of the record as opaque.  No part of this specification requires
 recursive resolvers to alter their behavior based on its contents, even if the contents
-are invalid.
+are invalid.  Recursive resolvers MAY validate the values of recognized
+SvcParamKeys and reject records containing invalid values.  However, for
+complex value types whose interpretation might differ between implementations
+(e.g. URIs), resolvers SHOULD limit validation to basic sanity checks.
```

Comment:
We should use something more descriptive/specific here.  
This wording is confusing.  (The term "sanity checks"
is also one that should avoid using from an inclusive
language perspective, if better and more descriptive terms
are available.)

Perhaps:

```suggestion
SvcParamKeys and reject records containing values 
which are invalid according to the SvsParam specification.
For complex value types whose interpretation might differ 
between implementations or have additional future
allowed values added (e.g. URIs or "alpn"), resolvers 
SHOULD limit validation to specified constraints.
```